forge-gui/res/cardsfolder/upcoming/flames_of_moradin.txt

```diff
@@ -0,0 +1,11 @@
+Name:Flames of Moradin
+ManaCost:2 R R
+Types:Sorcery
+A:SP$ Destroy | TargetMin$ 0 | TargetMax$ 3  | Cost$ 3 R | ValidTgts$ Artifact | SubAbility$ DBRepeatEach | SpellDescription$ Destroy up to three target artifacts. Conjure a duplicate of each nontoken artifact destroyed this way into your hand. The duplicates perpetually gain “You may pay {R} rather than pay this spell's mana cost" and "at the beginning of your end step, sacrifice this artifact."

```

Comment:
`TargetMax$ 3..| Cost$ 3 R | ` - think this cost is bad? Just remove?

Reply:
I think + `TgtPrompt$ Select up to three target artifacts`? or does Forge already parse this well?

Reply:
`“You` > `"You`